### DISCUSSION TITLE
can i add widget in window

### DISCUSSION THREAD

**Discussion by ghost** [TheAlgorithms/Python#8055] [Q&A]
can I add a python widget to windows home screen and desktop etc.

**Comment by Youssef-AMARZOU** [comment]
Honestly I never thought Python could do desktop widgets until I stumbled across pystray last year. Built a little weather thing for my taskbar in like twenty minutes and it actually worked. The trick is you gotta use system tray since Windows doesn't really let Python touch the home screen directly. If you're fancy you can wrap a Flask backend with Electron but that's overkill for most stuff. Just my two cents though, someone else might know a cleaner way.
